src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -211,6 +212,9 @@ function ReportActionItemMessageEdit(
         // eslint-disable-next-line react-hooks/exhaustive-deps -- this cleanup needs to be called only on unmount
     }, [action.reportActionID]);
 
+    // show the composer quickly after done editing for devices that hide the composer during editing.
```

Comment:
MInor wording adjustment: 

```suggestion
    // show the composer after editing is complete for devices that hide the composer during editing.
```